front/cypress/e2e/idea_new_page.cy.ts

```diff
@@ -51,7 +51,7 @@ describe('Idea new page', () => {
 
     cy.get('#e2e-idea-new-page');
     cy.get('#idea-form');
-
+    cy.contains('Add new idea').should('exist');
     // add a title and description
```

Comment:
Just need to merge in my recent formbuilder updates, and then I will add the test to make sure the title is correct for surveys as well. Will do tomorrow morning.